docs/playbook-reference/actions/scans.rst

```diff
@@ -63,6 +63,58 @@ To set custom tolerations or a nodeSelector update your ``generated_values.yaml`
         nodeSelector:
           nodeName: "your-selector"
 
+Customizing Efficiency Recommendations in the Robusta UI
+====================================================================================
+You can tweak KRR's recommendation algorithm to suit your environment using ``krr_args``.
```

Comment:
What are ``krr_args``?

I suggest "using the ``krr_args`` setting in Robusta's Helm chart."

Reply:
Done :heavy_check_mark: